.github/CODEOWNERS

```
@@ -6,6 +6,9 @@
 # team-specific repo segments
 /.github  @determined-ai/infrastructure
 /docs     @determined-ai/docs
+/master   @determined-ai/backend-notify
+/agent    @determined-ai/backend-notify
+/proto    @determined-ai/backend-notify
```

Use the non `-notify` group for ownership, and the -notify group for people who need CC'd.  The (barely-documented) idea is that `-notify` won't be configured to auto-assign a person (so groups can potentially just be notified without having to take action), and the "basename" group will actually autoassign to people as the team sees fit.

```suggestion
/master   @determined-ai/backend 
/agent    @determined-ai/backend 
/proto    @determined-ai/backend 
```

---

Perfect, thanks!